core/src/com/nerds/gamejam/screen/GameScreen.java

```diff
@@ -23,10 +23,10 @@ public GameScreen(GameJam game, Stage stage, Skin skin) {
 
     @Override
     public void show() {
-        Table table = new Table();
-        table.add(new Label("Hello there!", skin)).expand();
-        table.setFillParent(true);
-        stage.addActor(table);
+        ScrollingBackground scrollingBackground = new ScrollingBackground(new Texture("stars.png"));
```

Comment:
Seems weird to set the scrolling background again here - means it jumps to the initial position again - I think having the "hello there" placeholder was good enough given we'll be presumably replacing this with a character selection screen?

Reply:
boooo Daniel! We should have had a rule against all boring bastards